docs/HOWTO/90_Misc/Unpack_dvd.rst

```diff
@@ -0,0 +1,105 @@
+.. _howto-unpack-dvd:
+
+HOWTO Use unpack_dvd to Extract the Full OS RPM Package Set
+===========================================================
+
+The SIMP ISO provides a minimal set of packages.
+
+If you require additional OS packages, you can extract them from vendor ISOs using
+:program:`unpack_dvd`.  Additionally you can extract the extract the PXE boot
+TFTP boot files from the ISO to the rsync directories.
+
+Use option :code:`unpack_dvd --help` to see all options available for :program:`unpack_dvd`.
+
+Extract the OS Packages
+-----------------------
+
+:program:`Unpack_dvd` extracts the OS RPMs to :file:`/var/www/yum/<OperatingSystem>`
+in a directory named after the OS version it determines from files on the ISO.
+It then creates a link to the major version.  If version 7.6 is extracted,
+7 will be linked to 7.6.
+
+After extracting all the RPMs it will create or update a repository in the
+versioned directory.
+
+.. NOTE::
+
+   If unpack_dvd can only determine the major OS version from the files
+   on the ISO, :program:`unpack_dvd` will ask you to supply a more descriptive
+   version number using ``-v`` option.
+
+The following example will:
+
+* extract the RPMs to :file:`/var/www/yum/CentOS/7.8.2003`
+* create a repository in the above directory
+* link :file:`/var/www/yum/CentOS/7` to the above directory.
+
+#. Log on as ``simp`` and run :command:`sudo su - root`.
```

Comment:
If you install SIMP from RPM, there will not be a `simp` user.  How about just telling the user to login as root?